packages/notification-center/src/i18n/languages/es.ts

```diff
@@ -4,7 +4,7 @@ export const ES: ITranslationEntry = {
   translations: {
     notifications: 'Notificaciones',
     markAllAsRead: 'marcar todo como leido',
-    poweredBy: 'Energizado por',
```

Comment:
Could you add also the translation for settings? 🙏🏻

Reply:
done! missed that string!